### PR TITLE
Readme and pr template

### PR DESCRIPTION
**Pull request by loadingthedev**
### Update readme to get google oauth credentials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
 # LISTD Media
 
 An app that will allow users to create, share and watch lists of YouTube channels.
+
+## Setup
+###  Getting Google OAuth API Credentials
+
```
Comment (omar2205) [inline README.md:7]: Add - Move (rename) `.env.example` to `.env`

Comment (loadingthedev) [reply README.md:7]: ok i will update 😅

**Comment by w3cj** [comment]
Thanks! Can you move this from the README.md to CONTRIBUTING.md?

**Comment by loadingthedev** [comment]
sure i will move this to contribution.md

**Pull request merged**